src/api/viewSetup.ts

```diff
@@ -138,9 +134,11 @@ export class ViewSetup {
     if (!iModel.isClosed) {
       const selectRelevantCategories = `SELECT ECInstanceId FROM BisCore.SpatialCategory ${categoryCodes.length > 0 ? `WHERE CodeValue IN ('${categoryCodes.join("','")}')` : ""}`;
       for await (const row of iModel.query(selectRelevantCategories)) {
```

Comment:
Would serve as a better (more efficient) example if you did this with a single ECSql statement.

Reply:
Combined into single statement.